index.html

```diff
@@ -0,0 +1,156 @@
+                    </div><br>
+                      <div>
+                        <button class="btn-proj">See Project</button>
+                      </div>
+           </div>
+             <div class="container3">
+              <div>
+                <img class="portolio_img" src="third_img.png" alt="portfolio image">
+              </div>
+              <div>
+                <p class="tittle">Tonic</p>
+              </div>
+              <div>
+                <p class="words"><b class="canopy">CANOPY</b><img class="img" src="Counter.png" alt="count">Back End Dev<img class="img" src="Counter.png" alt="count">2015</p>
+              </div>
+                <div>
+                  <p class="paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                    <div>
+                        <button>html</button>
+                        <button>css</button>
+                        <button>javascript</button>
+                    </div><br>
+                      <div>
+                        <button class="btn-proj">See Project</button>
+                      </div>
+          <div class="container4">
+            <div>
+              <img class="portolio_img" src="forth_img.png" alt="portfolio image">
+            </div>
+              <div>
+                <p class="tittle">Tonic</p>
+              </div>
+                <div>
+                  <p class="words"><b class="canopy">CANOPY</b><img class="img" src="Counter.png" alt="counter">Back End Dev<img class="img" src="Counter.png" alt="counter">2015</p>
+                </div>
+                  <div>
+                    <p class="paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required </p>
+                  </div>
+                    <div>
+                        <button>html</button>
+                        <button>css</button>
+                        <button>javascript</button>
+                    </div><br>
+                      <div>
+                        <button class="btn-proj">See Project</button>
+                      </div>
+          </div>
+            <section class="about">
+       <h2 class="header">About <br> Myself</h2>
+          <p class="section">
+           I’m a software developer! I can help you build a product,feature or 
+           website Look through some of my work and experience! If you like what 
+           you see and have a project you need coded, don’t hestiate to contact me.
+          </p>
+           <p class="footer">LET CONNECT</p>
+            <div class="img">
+              <a href="#"><img src="twitter.svg" alt="twitter handle"></a>
+              <a href="#"><img src="github.jpg" alt="github handle"></a>
+              <a href="#"><img src="medium.jpg" alt="medium handle"></a>
+              <a href="#"><img src="angellist.jpg" alt="angellisthandle"></a>
+            </div>
```

Comment:
- [ ] As I have mentioned above, would be nice to add these links to `ul` `li` tags 🤗 :

```
<ul>
  <li><a href="#"><img src="twitter.svg" alt="twitter handle"></a></li>
  <li><a href="#"><img src="github.jpg" alt="github handle"></a></li>
  ...
</ul>
```

- [ ] These links are positioned slightly different from original template, I kindly suggest to fix margins and match with Figam template:

Figma design   | Your design
------------- | -------------
![image](https://user-images.githubusercontent.com/77038610/143862712-8bee6960-5a62-4e20-a9da-b6a8978806dd.png)  | ![image](https://user-images.githubusercontent.com/77038610/143862665-8e120f41-d11b-4e1a-95e1-23413b7b1380.png)

- [ ] Please try to match margins and paddings with Figma template ☀️